settings/config.lua

```diff
@@ -0,0 +1,15 @@
+-- Do not set listen for now so connector won't be
```

Comment:
Let's move the file to `settings/testdata` folder.

Reply:
Moved